doc/protocol.md

```diff
@@ -829,6 +829,119 @@ or pipe is done.
 Additionally, a "options" control message may be sent in this channel
 to change the "max-size" options.
 
+Payload: fsinfo
+----------------
+
+This channel is used to report information about a given path, and the file or
+a directory which may reside there, including the possibility of updates.  In
+the case of directories, information can also be reported about the (directly)
+contained entries.
+
+Additional "open" command options should be specified with a channel of
+this payload type:
+
+ * `path` (mandatory): a string
+ * `attrs` (mandatory): a list of strings, the list of attributes to be
+   reported for the requested file.  See below for the possibilities.
+ * `fnmatch`: (optional, default ''): if non-empty, this is a [`fnmatch`-style
+   pattern](https://docs.python.org/3/library/fnmatch.html) selecting the directory
+   entries to be reported.  Use `*` to report all files.  Can also be used to
+   implement search (`searc*`) or hide temporary files (`[!.]*`).
+ * `close-on-error` (optional, default: true): if the channel should close
+   (with `problem`) on failure to `open()` the named `path`
```

Comment:
We already discussed this a week ago and you told me, but I keep tripping over this: what happens with `false`? It gets explained two pages down (and not super easy to find). Perhaps this should just say "see below".

Reply:
I'll add a "see below".  Keep in mind that this is `protocol.md`, which is basically a reference guide for ourselves...

Reply:
This is gone -- this was confusing and also mostly redundant with `watch`.